kazuki-core-test/src/main/kotlin/com/anaplan/engineering/kazuki/core/SetExtension.kt

```diff
@@ -4,6 +4,8 @@ package com.anaplan.engineering.kazuki.core
 interface SetExtension<T> : Set<T> {
 }
 
+
+
```

Comment:
❓ Did you mean to have a tests for a pretty printable set extension as well?

Reply:
it's not very interesting so changed my mind :smile: